lib/view_model/trade_details_view_model.dart

```diff
@@ -90,6 +91,15 @@ abstract class TradeDetailsViewModelBase with Store {
           value: trade.provider.toString()));
     }
 
+    if (trade.provider.title == "ChangeNOW") {
```

Comment:
Unnamed constant. If `provider` in current context is `ExchangeProviderDescription` then compare with a static constant which you need instead of compare title strings.

Reply:
Like this? `if (trade.provider is ChangeNowExchangeProvider) {`

Reply:
Nope. `provider` in current context is `ExchangeProviderDescription` not `ExchangeProvider` or `ChangeNowExchangeProvider`.

Reply:
Implemented static comparison between `trade.provider` and `ExchangeProviderDescription.changeNow`